client/my-sites/stats/stats-tabs/tab.jsx

```diff
@@ -100,7 +100,12 @@ class StatsTabsTab extends Component {
 					{ hasPreviousData && (
 						<div className="stats-tabs__highlight">
 							<span className="stats-tabs__highlight-value" ref={ this.tooltipRef }>
-								{ formatNumber( value ) }
+								{ numberFormat( value, {
+									numberFormatOptions: {
+										notation: 'compact',
+										maximumFractionDigits: 1,
```

Comment:
A few more of these cases and I'll be returning to consider why we set a [default = 0 on `decimals`](https://github.com/Automattic/wp-calypso/blob/c783b2255af1ced8f6d089d8c39462a5ea0fdb67/packages/i18n-calypso/types/index.d.ts#L69), as this wouldn't be needed without that default.

It might also be worth abstracting "compact" functionality behind a `numberFormat` top-level option so it unifies these options across every case.